### PR TITLE
Lab04 forms

### PR DESCRIPTION
**Pull request by tektechnologies**
this branch is not lab four, it is infact updating netlify requirements.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
   },
   "devDependencies": {
     "eslint": "^3.19.0",
-    "eslint-plugin-react": "^7.1.0"
+    "eslint-plugin-react": "^7.1.0",
+    "node_env": "production"
```
Comment (dahlbyk) [inline package.json]: `node_env` does not belong in `devDependencies`

File: package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "npm run build",
```
Comment (dahlbyk) [inline package.json]: Recursion!

Comment (tektechnologies) [reply package.json]: What do you mean, that I was making it run.... @dahlbyk

File: package.json (hunk 1 above)
Comment (tektechnologies) [reply package.json]: I forget where the docs said to put that, in another Dependencies? 
@dahlbyk

**Comment by tektechnologies** [comment]
@selmaT273 would you do a review? maybe check the live versions, I think they may be working. @selmaT273

File: package.json (hunk 1 above)
Comment (tektechnologies) [reply package.json]: does it go in SCRipts. @dahlbyk

**Pull request merged**